source/default/_patterns/00-protons/non-printing/_spacing.scss

```diff
@@ -1,7 +1,13 @@
 // Custom spacing variables and mixins
+// You can add more entries to the $spacers map, should you need more variation.
 
-$spacing: (
-  s: 5px,
-  m: 25px,
-  l: 50px,
-);
+$spacer: 1rem !default;
+$spacers: () !default;
+$spacers: map-merge(
+  (
+    s: ($spacer * 0.3125), //5px
```

Comment:
Let's not stress putting in our old "s,m,l" versions. Let's just use the existing 1-5.